qiskit_experiments/tomography/basis/base_basis.py

```diff
@@ -0,0 +1,125 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Fitter basis classes for tomography analysis.
+"""
+from abc import ABC, abstractmethod
+from typing import Iterable, Optional
+import numpy as np
+from qiskit import QuantumCircuit
+
+
+class BaseFitterMeasurementBasis(ABC):
+    """Abstract base class for fitter measurement bases."""
```

Comment:
Can you elaborate the difference of `Fitter` and `Tomography` (especially the purpose of them)? It was hard to follow until I asked you offline.